plugins/teststeps/slowecho/slowecho.go

```diff
@@ -72,10 +71,6 @@ func (e *Step) ValidateParameters(params test.TestStepParameters) error {
 		return errors.New("missing 'sleep' field in slowecho parameters")
 	}
 
-	// no expression expansion here
-	if len(secStr.String()) != 1 {
-		return fmt.Errorf("invalid empty 'sleep' parameter: %v", secStr)
-	}
```

Comment:
It was just wrong. Due to this lines I was unable to run a `slowecho` with `sleep` higher than `9`.